notebooks/test_distributions.ipynb

```diff
@@ -0,0 +1,975 @@
+{
```

Comment:
<p>The failing distribution seems heavily skewed and Im not sure we will get a suitable fit without a transformation. Can we try log transform, or sq root transform or box-cox on the data here and see what gets us easier fits?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/optimal-stopping-point-tool/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Sure, that's something I can look at.